code/modules/events/vent_clog.dm

```diff
@@ -0,0 +1,116 @@
+/datum/round_event_control/vent_clog
+	name = "Clogged Vents: Normal"
+	typepath = /datum/round_event/vent_clog
+	weight = 10
+	max_occurrences = 3
+	min_players = 10
+
+/datum/round_event/vent_clog
+	announceWhen	= 1
+	startWhen		= 5
+	endWhen			= 35
+	var/interval 	= 2
+	var/list/vents  = list()
+	var/randomProbability = 1
+	var/reagentsAmount = 500
+	var/list/saferChems = list(/datum/reagent/water,/datum/reagent/carbon,/datum/reagent/consumable/flour,/datum/reagent/space_cleaner,/datum/reagent/consumable/nutriment,/datum/reagent/consumable/condensedcapsaicin,/datum/reagent/drug/mushroomhallucinogen,/datum/reagent/lube,/datum/reagent/glitter/pink,/datum/reagent/cryptobiolin,
```

Comment:
1. this list needs to be static or done on Initialize
2. I would much prefer if this used the existing CHEMFLAGS, and iterated over the reagents rather than having this random big list

Reply:
I'll revisit this after testing.
I may end up just removing the entire event if it runs like doo doo on the main box.